doc/src/site/sphinx/deployment/load_mojo.rst

```diff
@@ -308,4 +310,17 @@ To obtain the leaf node assignments, please first make sure to set ``withLeafNod
 ``withDetailedPredictionCol`` to true on your MOJO settings object. The leaf node assignments are now stored
 in the ``${detailedPredictionCol}.leafNodeAssignments`` column on the dataset obtained from the prediction.
 Please replace ``${detailedPredictionCol}`` with the actual value of your detailed prediction col. By default,
-it is ``detailed_prediction``.
\ No newline at end of file
+it is ``detailed_prediction``.
+
+Obtaining Stage Probabilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To obtain the stage results, please first make sure to set ``withStageResults`` and
+``withDetailedPredictionCol`` to true on your MOJO settings object. The stage results for regression and anomaly detection problems
+are stored in the ``${detailedPredictionCol}.stageResults`` on the dataset obtained from the prediction. The stage results for classification
+(binomial, multinomial) problems are stored under ``${detailedPredictionCol}.stageResults`` Please replace ``${detailedPredictionCol}``
```

Comment:
It seems that all binomial, multinomial, regression and anomaly store the stage results in `${detailedPredictionCol}.stageResults`. Can we simplify this sentence?

Reply:
Good catch, the second sentence is wrong. it should be ``${detailedPredictionCol}.stageProbabilities``